templates/component/name.component.js

```diff
@@ -5,6 +5,7 @@ angular.module('<%= scriptAppName %>')
     template: '<h1>Hello {{ $ctrl.message }}</h1>',
     bindings: { message: '<' },
     controller: function() {
-      this.message = 'World'
+      var vm = this;
+      vm.message = 'World';
```

Comment:
I just did as for all other component. But, perhaps, it's unnecessary. There is no styleguide for component anywhere.